### PR TITLE
Updated to latest version

### PR DESCRIPTION
**Pull request by matmel**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR updates `vega-cli` to its current latest version. It also tries to solve the issue explained in this detailed comment: https://github.com/altair-viz/altair_saver/issues/13#issuecomment-632963608. 

The TL;DR is that this package is currently broken with `altair_saver` conda package because it uses a local install instead of a global install of the node packages.

### CODE REVIEW

**Comment by matmel** [comment]
@conda-forge-admin, please rerender
